tests/testthat/test-quit_if_no_pacta_relevant_data.R

```diff
@@ -0,0 +1,51 @@
+test_that("`quit_if_no_pacta_relevant_data()` quits if no ABCD in non-interactive", {
+  skip_if_R_CMD_check()
+
+  tmp_dir <- withr::local_tempdir()
+  port_name <- "TestPort"
+  expect_null({
+    callr::r(
+      func = function(tmp_dir, port_name) {
+        devtools::load_all()
+        source(here::here("0_global_functions.R"))
+        port_with_no_data <- data.frame(has_asset_level_data = FALSE)
+        outputs_path <<- tmp_dir
+        portfolio_name_ref_all <<- port_name
+        log_path <<- outputs_path
```

Comment:
Ok so it looks like the function `quit_if_no_pacta_relevant_data()` actually requires the invisible arguments `outputs_path`, `portfolio_name_ref_all` and `log_path`, (I guess because of the `log_user_errors()` functuon also has those invisible dependencies). 

Changing `log_user_errors` is beyond the scope of this PR, but maybe it's good to make it obvious in the function trace of `quit_if_no_pacta_relevant_data()` that those objects are required...

Reply:
> Ok so it looks like the function `quit_if_no_pacta_relevant_data()` actually requires the invisible arguments `outputs_path`, `portfolio_name_ref_all` and `log_path`, (I guess because of the `log_user_errors()` functuon also has those invisible dependencies).

💯 

> Changing `log_user_errors` is beyond the scope of this PR, but maybe it's good to make it obvious in the function trace of `quit_if_no_pacta_relevant_data()` that those objects are required...

What exactly does it mean to "make it obvious in the function trace"?

Reply:
I mean adding these arguments directly to the function. I know it won't do much, but at least it will make it very clear when looking at the function that's it is necessary that those objects exist.

Reply:
I think that would cause more confusion than benefit... if we add those as arguments, whatever gets passed to them will not be used because `log_user_errors()` will still look in the global environment for them, not the function's local environment... so then it's doubling down on the confusion of where those values are coming from.

Reply:
Yup that is fair.